### PR TITLE
Movie details work

### PR DESCRIPTION
**Pull request by Dawid-Zieba**
Added styled for movie details,background and some responsiveness.

### CODE REVIEW

File: src/common/PersonTile/styled.js [hunk 1]
```diff
@@ -69,4 +70,52 @@ export const Name = styled.div`
...
+  border-radius: 5px;
+  text-align: center;
+  height: 100%;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.medium}) {
+    padding: 8px;
+  }
+`;
+export const Role = styled.span`
+  color: ${({ theme }) => theme.color.waterloo};
```
Comment (MateuszPiaszczyk) [inline src/common/PersonTile/styled.js:95]: you missed the "s" :) please replace with : color: ${({ theme }) => theme.colors.waterloo}; 👍

File: src/common/PersonTile/styled.js [hunk 2]
```diff
@@ -69,4 +70,52 @@ export const Name = styled.div`
...
+    css`
+      -webkit-line-clamp: 2;
+      max-height: 2.6em;
+    `}
 `;
+
+
+export const Person = styled.article`
+  padding: 16px;
+  background-color: ${({ theme }) => theme.color.white};
```
Comment (MateuszPiaszczyk) [inline src/common/PersonTile/styled.js:84]: you missed the "s" :) please replace with : color: ${({ theme }) => theme.colors.white} 👍

**Review by MateuszPiaszczyk — APPROVED** [review]
Great Job man :)

File: src/common/PersonTile/styled.js (hunk 1 above)
Comment (Dawid-Zieba) [reply src/common/PersonTile/styled.js]: Thanks! My keyboard has been failing me lately ;c

**Pull request merged**